docs/options.md

```diff
@@ -354,14 +368,29 @@ CIBW_BEFORE_ALL: make -C third_party_lib
 CIBW_BEFORE_ALL_LINUX: yum install -y libffi-dev
 ```
 
+Note that manylinux2_24 builds occur inside a Debian9 docker, where
+manylinux2010 and manylinux2014 builds occur inside a CentOS one. So for
+`manylinux2_24` the `CIBW_BEFORE_ALL_LINUX` command must use `apt-get -y`
+instead.
+
```

Comment:
This is the added blurb, it kind of got lost in the reflow.

Reply:
Yeah, GH's ignore whitespace doesn't work on reflowing things.

Reply:
What about the whitespace? If we want to do this, we should probably be consistent (and do it in a different PR) ?

I can push a commit to minimize the diff, if you want?

Reply:
Alright, I did so (together with a minor typo). Feel free to kick out the last commit (67903ae0a2e7e5d34f864ded03bd66ea8cd964ea) again, if you disagree :-)